server/controllers/admin/userManagementController.ts

```diff
@@ -11,6 +13,15 @@ export default class UserController {
       res.render('admin/users/index', { pageHeading: 'User management dashboard', users })
     }
   }
+
+  show(): TypedRequestHandler<Request, Response> {
```

Comment:
Pretty minor, but this feels like an edit action rather than a show action to me